apps/pipelines/management/commands/import_pipeline.py

```diff
@@ -0,0 +1,43 @@
+import json
```

Comment:
It wouldn't be hard to take this code and create, say 1 or 2 default pipelines for each team where pipelines is being enabled. Those can serve as examples.

Reply:
I just made this because it was the quickest way to import one of the pipelines from prod. But I can imagine an import / export / copy feature being useful.